src/css/main.css

```diff
@@ -0,0 +1,280 @@
+/* ・ｽu・ｽ・ｽ・ｽE・ｽU・ｽﾔの表・ｽ・ｽ・ｽ・ｽ・ｽﾙに対会ｿｽ・ｽ・ｽ・ｽ驍ｽ・ｽﾟ、・ｽﾜゑｿｽ・ｽ]・ｽ・ｽ・ｽ・ｽ・ｽS・ｽ・ｽ0・ｽﾉゑｿｽ・ｽ・ｽ */
```

Comment:
Please avoid using Japanese comments. Since it will be later read by OSS code contributor who does not understand Japanese.